build/publish/PublishDebian.targets

```diff
@@ -1,23 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <PropertyGroup>
-    <SdkDebianPackageName>dotnet-sdk-$(SdkVersion)</SdkDebianPackageName>
-  </PropertyGroup>
-
-  <Target Name="PublishDebFilesToDebianRepo"
-          DependsOnTargets="SetupDebProps;"
-          Condition=" '$(IsBuildingAndPublishingAllLinuxDistrosNativeInstallers)' == 'true' AND '$(SkipPublishToDebianRepo)' != 'true' AND '$(IslinuxPortable)' != 'true' " >
-    <Error Condition="'$(REPO_USER)' == ''" Text="REPO_USER must be set as a MsBuild Property variable for debian publishing." />
-    <Error Condition="'$(REPO_PASS)' == ''" Text="REPO_PASS must be set as a MsBuild Property variable for debian publishing." />
-    <Error Condition="'$(REPO_SERVER)' == ''" Text="REPO_SERVER must be set as a MsBuild Property variable for debian publishing." />
-
-    <UploadToLinuxPackageRepository Condition=" '%(LinuxDistrosNeedNativeInstaller.RepositoryId)' != '' "
```

Comment:
Can we remove the `UploadToLinuxPackageRepository` task code as well?

Reply:
Done...